tracepointdebug/probe/log_point_manager.py

```diff
@@ -92,6 +95,21 @@ def expire_log_point(self, log_point):
                 self._log_points.pop(log_point_id).remove_log_point()
 
     def publish_event(self, event):
+        try:
+            if self.broker_manager._log_data_redaction_callback and isinstance(event, LogPointEvent):
+                log_redaction = {
+                    "file_name": event.file,
+                    "line_no": event.line_no,
+                    "method_name": event.method_name,
+                    "log_message": event.log_message,
+                }
+                self.broker_manager._log_data_redaction_callback(log_redaction)
```

Comment:
How do we expect logpoint callback to be implemented? @gokhan721 @boroskoyo 

Let's say that I want to mask passwords in the log message? How can I achieve it in log data redaction callback? As far as I see, it get raw string log message. So how user will implement a callback to mask passwords in the log message?